src/style.css

```diff
@@ -1,3 +1,79 @@
+html {
+  font-family: Roboto, sans-serif;
+
+  --blue-word: #00acdc;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Consider using the universal selector (*) to apply 'box-sizing: border-box;' to all elements, pseudo-elements, and before/after elements. This ensures consistent box-sizing across the entire document.